draft-ietf-quic-recovery.md

```diff
@@ -135,6 +135,12 @@ Crypto Packets:
 : Packets containing CRYPTO data sent in Initial or Handshake
   packets.
 
+Out-of-order Packets:
+
+: Packets that do not increase the largest received packet number for its
+  packet number space by exactly one. Packets arrive out of order
+  when earlier packets are lost or delayed.
```

Comment:
I'd be inclined to drop earlier because it's potentially confusing.
```suggestion
  when packets are lost or delayed.
```

Reply:
I don't know... I think it might read better with 'earlier' included. I don't think it's confusing.